.github/actions/wasi/base/action.yml

```diff
@@ -0,0 +1,16 @@
+name: "Ubuntu base dependencies"
+
+runs:
+  using: "composite"
+  steps:
+    - name: Install system dependencies
+      shell: bash
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y ninja-build wget
+    - name: Install CMake
+      shell: bash
+      run: |
+        wget -nv -O ./cmake_install.sh https://github.com/Kitware/CMake/releases/download/v3.28.0-rc5/cmake-3.28.0-rc5-linux-x86_64.sh
+        sudo bash ./cmake_install.sh --skip-license --prefix=/usr --exclude-subdir
+        rm -f ./cmake_install.sh
```

Comment:
Could we use cmake which provide by ubuntu? 
If yes, then we can just re-use action from linux/base

Reply:
Good mention. I thought a newer version of CMake was needed, but it was not. 
Using the ubuntu-provided cmake now :)